src/visitpy/visit_utils/src/builtin/__init__.py

```diff
@@ -12,4 +12,6 @@
 
 from .evalfuncs import *
 from .writescript import WriteScript
-from .convert2to3 import ConvertPy2to3
\ No newline at end of file
+from .convert2to3 import ConvertPy2to3
+from .convert2to3 import GetAutoPy2to3
+from .convert2to3 import SetAutoPy2to3
```

Comment:
It says there is no newline at the end of the file. Don't know if this needs fixing, but I'm pointing it out.

Reply:
added newline